packages/fxa-payments-server/src/index.tsx

```diff
@@ -59,6 +60,10 @@ function locationReload() {
   window.location.reload();
 }
 
+function matchMedia(query: string) {
+  return window.matchMedia(query);
```

Comment:
If this was `return window.matchMedia(query).matches` then the type of this function could just be `(query: string) => boolean` and not drag in all of `MediaQueryList`

Reply:
that would greatly simplify things, thank you